src/specs/api.yaml

```diff
@@ -314,11 +314,32 @@ paths:
         '200':
           description: OK
       description: Deletes a plot and table for a given experiment with the data specified.
+  '/experiments/{experimentId}/pipelines':
+    parameters:
+      - schema:
+          type: string
+        name: experimentId
+        in: path
+        required: true
+    post:
+      summary: Create a new pipeline for taks execution
+      operationId: createNewPipeline
+      x-eov-operation-id: 'pipelines#create'
+      x-eov-operation-handler: routes/pipelines
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties: {}
+      description: This path will create a new pipeline that can run a state machine with different bioinformatics tasks.
```

Comment:
where do we supply the tasks that we want the statemachine to run?

Reply:
We don't at this point, the ticket's scope does not extend to that, see https://biomage.atlassian.net/browse/BIOMAGE-512